src/project/qml/MuseScore/Project/ProjectPropertiesDialog.qml

```diff
@@ -0,0 +1,317 @@
+                    navigation.panel: headerProperties.navigationPanel
+                    navigation.row: 3
+                }
+
+                StyledTextLabel {
+                    text: qsTrc("project", "API-Level:")
+                    font: ui.theme.bodyBoldFont
+                }
+
+                TextInputField {
+                    id: apiLevel
+
+                    Layout.preferredWidth: 100
+                    currentText: projectPropertiesModel.apiLevel
+                    readOnly: true
+
+                    navigation.name: "APILevelTextField"
+                    navigation.panel: headerProperties.navigationPanel
+                    navigation.row: 4
+                }
+
+                Item {      // To mimic a flatbutton so that it aligns with the open folder button
+                    Layout.preferredWidth: 30
+                    Layout.preferredHeight: 30
+                }
+            }
+        }
+
+        SeparatorLine {}
+
+        StyledListView {
+            id: propertiesListView
+
+            Layout.fillHeight: true
+            width: root.contentWidth
+            spacing: 4
+
+            property NavigationPanel navigationPanel: NavigationPanel {
+                name: "ProjectPropertiesListPanel"
+                section: root.navigationSection
+                direction: NavigationPanel.Both
+                order: 2
+                onActiveChanged: function(active) {
+                    if (active) {
+                        root.forceActiveFocus()
+                    }
+                }
+            }
+
+            model: projectPropertiesModel
+
+            delegate: RowLayout {
+                width: propertiesListView.width
+                spacing: 8
+
+                StyledTextLabel {
+                    Layout.preferredWidth: 110
+                    text: model.propertyName
+                    font: ui.theme.bodyBoldFont
+                    horizontalAlignment: Text.left
+                    visible: model.isStandardProperty
+                }
+
+                TextInputField {
+                    Layout.preferredWidth: 110
+                    currentText: model.propertyName
+                    visible: !model.isStandardProperty
+                    hint: qsTrc("project", "Property")
+                    onCurrentTextEdited: function(newTextValue) {
+                        model.propertyName = newTextValue
+                    }
+
+                    navigation.name: model.propertyName + "PropertyName"
+                    navigation.panel: propertiesListView.navigationPanel
+                    navigation.column: 3
+                    navigation.row: model.index
+                }
+
+                TextInputField {
+                    Layout.fillWidth: true
+                    currentText: model.propertyValue
+                    hint: model.isStandardProperty ? "" : qsTrc("project", "Value")
+                    onCurrentTextEdited: function(newTextValue) {
```

Comment:
Please bring the code to the standard(as in the project)
1. anchors
2. settings of local properties
3. navigation(with its signal handlers)
4. other signal handlers

Add new lines between them